security/aws/iam_no_root_access_keys/README.md

```diff
@@ -0,0 +1,54 @@
+# AWS Root Access Keys
+
+## What it does
+
+The 'root' user account is the most privileged user in an AWS account. AWS Access Keys provide programmatic access to a given AWS account. It is recommended that all access keys associated with the 'root' user account be removed. This policy checks the Root account for access k. An incident is created if one or more are present.
```

Comment:
This policy checks the Root account for an access key. An incident is created if one or more access keys are present.

Reply:
Will fix and push update shortly.